src/main/resources/org/jenkinsci/plugins/workflow/steps/ReadFileStep/help.html

```diff
@@ -1,3 +1,10 @@
 <div>
     Reads a file from a relative path (with root in current directory, usually workspace) and returns its content as a plain string.
 </div>
+<br>
+<p>Example:
+<pre><code>
+   readFile file: 'zorg.txt',
```

Comment:
Adds nothing beyond what snippet generator shows, since this is not showing how to assign the value to a variable and use it for some purpose.